rotkehlchen/exchanges/coinbase.py

```diff
@@ -285,36 +389,48 @@ def _api_query(
             options: dict[str, Any] | None = None,
             ignore_pagination: bool = False,
     ) -> list[Any]:
-        """Performs a coinbase API Query for endpoint
+        """
```

Comment:
Please @vijithreddy this is the 3rd time I ask you.

https://github.com/rotki/rotki/pull/7903/files#r1593544499

 Undo this formatting change. There is no need for it.

Reply:
I didn't realize I formatted by adding a new line. Apologies, reverting the change

Reply:
Issued a PR to rever the formatting change